components/ILIAS/OpenIdConnect/classes/class.ilOpenIdAttributeMappingTemplates.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+class ilOpenIdAttributeMappingTemplate
+{
+    final const OPEN_ID_CONFIGURED_SCOPES = 'auth_oidc_configured_scopes';
+
+    /**
+     * @param array{int, string} $additional_scopes
+     * @return array{string, string}
```

Comment:
This syntax is not correct, maybe my remark was a bit misleading:

Try to provide the array shape according to: https://phpstan.org/writing-php-code/phpdoc-types#array-shapes (struct-like arrays) and https://phpstan.org/writing-php-code/phpdoc-types#general-arrays (for maps/sets/lists etc.).

In both cases `<` and `>` should be used here (and most probably also in other places).